pkg/skaffold/util/util.go

```diff
@@ -107,6 +107,13 @@ func ExpandPathsGlob(workingDir string, paths []string) ([]string, error) {
 	return ret, nil
 }
 
+// HasMeta reports whether path contains any of the magic characters
+// recognized by filepath.Match.
+// This is a copy of filepath/match.go's hasMeta
+func HasMeta(path string) bool {
```

Comment:
nit: Meta -> WildCard

Reply:
I think its probably best to just use the name from the stdlib

Reply:
Not all of these at wildcards also